models/user.go

```diff
@@ -558,7 +558,7 @@ func (u *User) GetOrgRepositoryIDs() ([]int64, error) {
 	return ids, x.Table("repository").
 		Cols("repository.id").
 		Join("INNER", "team_user", "repository.owner_id = team_user.org_id").
-		Join("INNER", "team_repo", "(team_user.team_id = team_repo.team_id AND repository.id = team_repo.repo_id) OR repository.is_private != ?", true).
```

Comment:
Please remove `, true` since it is not used anymore

Reply:
Actually it is used. It's not possible to write `!= 0` because it doesn't work on all databases. You will see that pgsql fails if we use that.